src/Python/qsharp-core/qsharp/tests/test_iqsharp.py

```diff
@@ -6,11 +6,11 @@
 
 def test_simulate():
     """
-    Checks that a simple simulate works correctly
+    Checks that a simple simulate works correctly, both using callable() and 
+    callable.simulate()
     """
     from Microsoft.Quantum.SanityTests import HelloQ
-    r = HelloQ.simulate()
-    assert r == ()
+    assert HelloQ() == HelloQ.simulate() == ()
```

Comment:
Would it be worth also adding a line here using a simple operation that takes arguments (e.g. `HelloAgain`) to verify that arguments are handled correctly in both cases?

Reply:
Sure